10/alpine/docker-entrypoint.sh

```diff
@@ -107,15 +107,17 @@ docker_verify_minimum_env() {
 
 		EOWARN
 	fi
-	if [ -z "$POSTGRES_PASSWORD" ] && [ 'trust' != "$POSTGRES_HOST_AUTH_METHOD" ]; then
+	if [ -z "$POSTGRES_PASSWORD" ] && ! { [ 'trust' = "$POSTGRES_HOST_AUTH_METHOD" ] || [ 'reject' = "$POSTGRES_HOST_AUTH_METHOD" ]; }; then
 		# The - option suppresses leading tabs but *not* spaces. :)
 		cat >&2 <<-'EOE'
 			Error: Database is uninitialized and superuser password is not specified.
 			       You must specify POSTGRES_PASSWORD to a non-empty value for the
 			       superuser. For example, "-e POSTGRES_PASSWORD=password" on "docker run".
 
 			       You may also use "POSTGRES_HOST_AUTH_METHOD=trust" to allow all
-			       connections without a password. This is *not* recommended.
+			       connections without a password, or "POSTGRES_HOST_AUTH_METHOD=reject" to
```

Comment:
I'm unsure whether this should be documented here. If you're looking for this detailed of a configuration, you're probably better served by the dockerhub documentation than a log message.

More than happy to remove this if desired.